src/juv/_run_replace.py

```diff
@@ -1,21 +1,32 @@
 from __future__ import annotations
 
 import os
+import platform
 import signal
 import subprocess
 import sys
 
 from uv import find_uv_bin
 
+IS_WINDOWS = platform.system()
```

Comment:
Could we be more explicit here? I wonder about:

```suggestion
IS_WINDOWS = sys.platform.startswith("win")
```

and then remove the unecessary `platform` import.

Reply:
oh neat! I didn't realize sys had that functionality... and what I had there wasn't doing what I wanted anyway, oops, that's totally on me! I intended to put `platform.system=="Windows"`  :-/ I like your suggestion better either way!